Sources/StructuredData.swift

```diff
@@ -24,189 +70,396 @@ public protocol StructuredDataSerializer {
     func serialize(_ structuredData: StructuredData) throws -> Data
 }
 
+extension Optional where Wrapped: StructuredDataRepresentable {
+    public var structuredData: StructuredData {
+        switch self {
+        case .some(let wrapped): return wrapped.structuredData
+        case .none: return .null
+        }
+    }
+}
+
+extension Optional: StructuredDataFallibleRepresentable {
```

Comment:
Ohhh. The `a` has the `æ` noise. I feel like most people will read it like `fall` and be confused but maybe not.